crates/tactical-server/src/main.rs

```diff
@@ -0,0 +1,217 @@
+//! Tactical Server - Minimal Lightyear game server
+//!
+//! Simple flow:
+//! 1. Started by spawner with mission_id arg
+//! 2. Calls tactical_server_ready reducer with connection info
+//! 3. Runs game for clients
+//! 4. Calls commit_mission reducer on timeout/exit
+
+use std::net::{IpAddr, Ipv4Addr, SocketAddr};
+use std::time::Duration;
+
+use adventure_simulator_net::prelude::*;
+use adventure_simulator_net::protocol::WebTransportCertificateSettings;
+use bevy::prelude::*;
+use clap::Parser;
+use tracing::{error, info};
+
+/// Mission timeout in seconds
+const MISSION_TIMEOUT_SECS: u64 = 120;
+
+#[derive(Parser, Debug, Clone)]
+#[command(name = "tactical-server")]
+#[command(about = "Tactical mission server for Adventure Simulator")]
+struct Args {
+    /// Port to listen on
+    #[arg(long, default_value = "6000")]
+    port: u16,
+
+    /// Unique mission instance ID
+    #[arg(long)]
+    mission_id: String,
+
+    /// Scene key (e.g., "town_a", "town_b")
+    #[arg(long)]
+    scene_key: String,
+
+    /// SpacetimeDB URL (e.g., http://localhost:3000)
+    #[arg(long, default_value = "http://localhost:3000")]
+    spacetimedb_url: String,
+
+    /// SpacetimeDB module name
+    #[arg(long, default_value = "strategic-stdb-module")]
+    spacetimedb_module: String,
+
+    /// Public host for clients to connect to
+    #[arg(long, default_value = "localhost")]
+    public_host: String,
+}
+
+fn main() {
+    tracing_subscriber::fmt()
+        .with_env_filter(
+            tracing_subscriber::EnvFilter::from_default_env()
+                .add_directive("tactical_server=info".parse().unwrap())
+                .add_directive("bevy_app=warn".parse().unwrap())
+                .add_directive("bevy_ecs=warn".parse().unwrap()),
+        )
+        .init();
+
+    let args = Args::parse();
+    info!("Starting tactical server for mission {}", args.mission_id);
+    info!("Scene: {}, Port: {}", args.scene_key, args.port);
+
+    // Notify SpacetimeDB that we're ready (blocking before app starts)
+    let rt = tokio::runtime::Runtime::new().unwrap();
+    if let Err(e) = rt.block_on(notify_server_ready(&args)) {
+        error!("Failed to notify SpacetimeDB: {}", e);
+        std::process::exit(1);
+    }
```

Comment:
Oof, bevy should have its own async runtime ? So let's refactor this into a startup system that calls [bevy::tasks::block_on](https://docs.rs/bevy/latest/bevy/tasks/fn.block_on.html) (or just move it to `setup_server` system). Tbh I haven't used it, but I think it should work in this case ?